### PR TITLE
Add GOVERNANCE file

### LINKED ISSUE #1256: Create GOVERNANCE file
Let's document how the project is governed today, not changing anything yet, in a GOVERNANCE.md file. Over time we can iterate and improve this to make the project more transparent and inclusive.

* https://github.com/opengovernance/opengovernance.dev has examples of governance files from other projects. 
* https://sustainers.github.io/governance-readiness/ also has good questions to ask when writing the document

### PR DESCRIPTION
**Pull request by carolynvs**
# What does this change
This explains how the project governs itself today without attempting to introduce any language that we didn't already have existing. Over time we can iterate and improve upon it in follow-on pull requests. I want to get this in without modifying our existing goverance for our inclusion into the CNCF.

# What issue does it fix
Closes #1256 

# Notes for the reviewer
N/A

# Checklist
- [ ] Unit Tests
- [x] Documentation
- [ ] Schema (porter.yaml)

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+This document outlines how the Porter project governs itself. The Porter project
+consists of all of the repositories in the https://github.com/getporter
+organization. 
+
+Everyone who interacts with the project must abide by our [Code of Conduct]. 
+
+## Legal
+
+The Porter project is in the CNCF Sandbox. ⛱
```
Comment (vdice) [inline GOVERNANCE.md:11]: optional/idea: Do we want to link a CNCF Sandbox landing/info page?

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+* Contributor
+* Maintainer
+* Admin
+
+The [OWNERS] defines the current maintainers of the project.
+
+## Release Process
+
+Maintainers create the next release when the main branch is stable and related
+groupings of work is complete. We do not have a fixed cadence and prefer release
```
Comment (vdice) [inline GOVERNANCE.md:27]:
```suggestion
groupings of work is complete. We do not have a fixed cadence and prefer to release
```

**Review by vdice — APPROVED** [review]
A few minor notes; otherwise LGTM!

**Review by dev-drprasad — APPROVED** [review]
LGTM!

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+## Roles
+
+We have a [Contribution Ladder] that explains the capabilities and
+responsibilities of each role:
+
+* Contributor
+* Maintainer
+* Admin
+
+The [OWNERS] defines the current maintainers of the project.
```
Comment (iennae) [inline GOVERNANCE.md:22]: The OWNERS file?

File: GOVERNANCE.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+Everyone who interacts with the project must abide by our [Code of Conduct]. 
+
+## Legal
+
+The Porter project is in the [CNCF Sandbox](https://www.cncf.io/sandbox-projects/). ⛱
+
+## Roles
+
+We have a [Contribution Ladder] that explains the capabilities and
```
Comment (iennae) [inline GOVERNANCE.md:15]: Somewhere here should say explicitly that the contributor ladder includes the steps to progress to different roles (in other words, that this is the place to find out how to become a maintainer).

Comment (iennae) [reply GOVERNANCE.md:15]: Right now this makes it sound like the contribution ladder file just explains what the different roles and responsibilities are.

**Review by iennae — APPROVED** [review]
LGTM. I did add some comments but not required updates.

**Comment by carolynvs** [comment]
Thanks @iennae I've incorporated your feedback into the PR. I will merge as soon as I have Jeremy's 👍. After that we can improve as we go in follow-on PRs.

Apologies for the speed, just trying to put a bow on the CNCF transition stuff.

**Review by jeremyrickard — APPROVED** [review]
To me, Looks Good.

/lgtm

**Pull request merged**